packages/components/hub/ProfileButton.tsx

```diff
@@ -71,7 +71,7 @@ export const ProfileButton: React.FC<{
         }
       >
         <SecondaryButtonOutline
-          size="XL"
+          size="M"
```

Comment:
This change didn't create any regression on other components that use it ?
If you don't know please take care of that 👍